tools/c7n_gcp/c7n_gcp/filters/essential_contacts.py

```diff
@@ -0,0 +1,74 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Essential Contacts suppport for GCP organization
+"""
+from c7n.filters.core import ValueFilter
+from c7n.utils import local_session, type_schema
+from c7n_gcp.resources.resourcemanager import Organization
+import jmespath
+
+
+@Organization.filter_registry.register('essential-contacts')
```

Comment:
resource specific filters should go in the python module that defines the resource, the filters package is for filters that apply across resource types.

Reply:
Thx, Moved to appropriate resource module.